src/index.js

```diff
@@ -3,18 +3,29 @@ import { BrowserRouter } from 'react-router-dom';
 import ReactDOM from 'react-dom';
 import './Styles/base.scss';
 import App from './Components/App';
-import reportWebVitals from './reportWebVitals';
+import {
```

Comment:
Thank you for getting us set up on the initial set up for Apollo client and GraphQL